exchanges/huobi/huobi.go

```diff
@@ -64,6 +175,1637 @@ type HUOBI struct {
 	AccountID string
 }
 
+// Futures Contracts
+
+// FGetContractInfo gets contract info for futures
+func (h *HUOBI) FGetContractInfo(symbol, contractType, code string) (FContractInfoData, error) {
+	var resp FContractInfoData
+	params := url.Values{}
+	if symbol != "" {
+		params.Set("symbol", symbol)
+	}
+	if common.StringDataCompare(validContractTypes, contractType) {
```

Comment:
return error if contract type isn't valid and is supplied

Reply:
It isn't a requirement for the endpoint. One could send an empty string. Also if they send an incorrect contract type but with a correct symbol it still works.